Makefile

```diff
@@ -428,6 +432,9 @@ mipsel.deb linux_mipsel.tar.gz: export GOARCH := mipsle
 riscv64.deb riscv64.rpm linux_riscv64.tar.gz: export GOOS := linux
 riscv64.deb riscv64.rpm linux_riscv64.tar.gz: export GOARCH := riscv64
 
+riscv64.deb riscv64.rpm linux_loongarch64.tar.gz: export GOOS := linux
+riscv64.deb riscv64.rpm linux_loongarch64.tar.gz: export GOARCH := loong64
```

Comment:
I think you copy and pasted here, but did not update the riscv64 to loongarch64 in all places.

However, let's keep this to the tar.gz for now.

Reply:
It may be difficult to verify in a physical machine, but there are several ways to simulate Loongarch on the X86_64 physical machine：
1. You can use it on x86 machine with docker run -it --rm merore/archlinux-loongarch64.
2. Use [qemu-user-static](https://hub.docker.com/r/multiarch/qemu-user-static)
The above two methods use the different forms of QEMU to simulate. The specific use method is on the official website. Of course, if you want to know how to use any of the above methods, I also have some shallow experience. We can communicate with each other.
About merore/archlinux-loongarch64:
- Boot an Loongarch64-based Arch Linux with qemu-system-loongarch64
- The default user/password is root/loongarch64
- All dependcies required by criu build and test are pre-installed
- You must configure your own dependence on the environment, Golang's installation package is located at http://ftp.loongnix.cn/toolchain/golang/go-1.20/abi2.0/go1.20.2.linux-loong64.tar.gz
About qemu-user-static:
- Loongarch has received the official support of QEMU, so you can download [qemu-loongarch-static](https://github.com/multiarch/qemu-user-static/releases/download/v7.2.0-1/qemu-loongarch64-static)
- If you want to use the Golang image of Loongarch, you can access lcr.looongnix.cn
-  If you want to build it yourself, lcr.loongnix.cn/library/debian:sid is the most suitable basic image

Finally, thank you for your patience !

Reply:
Ok , thank you very much! 😊